zhaquirks/tuya/ts0601_sensor.py

```diff
@@ -15,6 +15,17 @@
     .add_to_registry()
 )
 
+
+(
+    TuyaQuirkBuilder("_TZE200_s1xgth2u", "TS0601")  # Nedis ZBSC30WT
+    .tuya_temperature(dp_id=1, scale=10)
+    .tuya_humidity(dp_id=2, scale=100)
```

Comment:
I forgot that the default scale for humidity is 100, we can just add the manufacturer and model, using `applies_to` to the quirk below.

Reply:
good catch. thanks - much better now.